apps/frontend/src/pages/meeting-details/meeting-details.tsx

```diff
@@ -134,8 +152,15 @@ const MeetingDetails: React.FC = () => {
 			<div className={styles["meeting-details"]}>
 				<div className={styles["meeting-details__header"]}>
 					<h1 className={styles["meeting-details__title"]}>
-						Meeting #{meeting.id} |{" "}
-						{formatDate(new Date(meeting.createdAt), "D MMMM hA")}
+						<span>Meeting #{meeting.id}</span>
+						<span className={styles["meeting-details__date"]}>
+							{formatDate(new Date(meeting.createdAt), "D MMMM hA")}
+						</span>
+
+						<MeetingStatusBadge
+							className={styles["meeting-details__status"] ?? ""}
```

Comment:
Does it work like this?
```suggestion
							className={styles["meeting-details__status"]}
```